src/style.css

```
@@ -0,0 +1,65 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.body {
+  font-family: Avenir, sans-serif;
+}
+  
+.search_bar {
+  height: 70px;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  background: url(images/Search.svg) no-repeat 26px 25px, #fff;
+  background-size: 19px;
+  padding-left: 62px;
+}
+
+.small_search_bar {
+  height: 42px;
+  width: 100%;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  background: url(images/Search.svg) no-repeat 13px 15px, #fff;
+  background-size: 11px;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  padding-left: 33px;
+}
+
+.small_search_bar:hover {
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.search_bar::placeholder {
+  font-family: Avenir, sans-serif;
```

move font-family under `html` selector and let inputs inherit it 
```suggestion
  font-family: inherit;
```

---

is not fixed